Makefile

```diff
@@ -478,6 +478,24 @@ version: $(VERSRC)
 # This rule triggers re-generation of version files specified if Makefile changes.
 $(VERSRC): Makefile
 	VERSION=$(VERSION) $(MAKE) -f version.mk setver
+	$(MAKE) update-api-module-path
+
+# This rule updates the api module path to be in sync with the current api release version.
+# e.g. github.com/gravitational/teleport/api/vX -> github.com/gravitational/teleport/api/vY
+# it will skip updating if:
+#  1. A suffix is present in the version - e.g. "-alpha"
+#  2. The major version hasn't changed - e.g. 7.0.0 -> 7.1.0
+.PHONY: update-api-module-path
+update-api-module-path: MAJOR_VERSION := $(shell echo "$(VERSION)" | cut -d"." -f1)
+update-api-module-path: VERSION_SUFFIX := $(shell echo "$(MAJOR_VERSION)" | awk '$$1 >= 2 {print "\\/v"$$1}')
+update-api-module-path:
+	test "$(VERSION)" && echo "$(VERSION)" | grep -vq "-" && head -1 api/go.mod | grep -vq "$(MAJOR_VERSION)"
```

Comment:
@andrejtokarcik Can you help me fix this line? There are two problems
 - The second check for Major version doesn't work
 - When this line fails, such as when "-alpha" is present, it returns exit code 1. So when `make` calls `make version` and that calls `make update-api-module-path`, this line causes `make` to fail. Can this be easily altered to instead exit gracefully?